core/kazoo_config/src/kapps_config.erl

```diff
@@ -853,7 +853,7 @@ migrate_config_setting(UpdatedFrom, Removed, To) ->
     case add_config_setting(To, Removed) of
         {'ok', UpdatedTo} ->
             {'ok', _} = kz_datamgr:save_doc(?KZ_CONFIG_DB, UpdatedTo),
-            {'ok', _} = kz_datamgr:save_doc(?KZ_CONFIG_DB, UpdatedFrom),
+            {'ok', _} = kz_datamgr:ensure_saved(?KZ_CONFIG_DB, UpdatedFrom),
```

Comment:
what are you talking about? It's the config migration code.
Anyway, I'll implement the process I described on slack